.github/workflows/tox-pytest.yml

```diff
@@ -52,8 +54,8 @@ jobs:
 
       - name: Log SQLite3 version
         run: |
-          conda run -n pudl-test which sqlite3
```

Comment:
Because the environment is activated system-wide during the provisioning of the runner, and we're using `bash -l {0}` which somehow make the shell realize that the conda environment is there, without needing to jump through the hoops of activation / .profile manipulations.  I'm not 100% clear on that part.